### PR TITLE
Add function that calls Youtube Data API to get items from a given playlist-id.

### PR DESCRIPTION
**Pull request by josefj1519**

### CODE REVIEW

File: src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java [hunk 1]
```diff
@@ -104,4 +111,34 @@ public void scrapeChannelUploadPlaylist_IOException() throws IOException {
       assertThrows(IOException.class, () -> scraper.scrapeChannelUploadPlaylist(IOEXCEPTION));
   }
 
+  @Test
+  public void getPlaylistItems_nonExistentUploadId() throws IOException {
+      mockPlaylistResponse = new PlaylistItemListResponse();
+      when(mockListPlaylistItems.execute()).thenReturn(mockPlaylistResponse.setItems(null));
+      Optional<List<PlaylistItem>> actual = scraper.getPlaylistItems(UPLOAD_ID_THAT_DOES_NOT_EXIST);
+      assertEquals(false, actual.isPresent());
```
Comment (margaretchan) [inline src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java]: Consider using the Hamcrest assert library as Sam mentioned [here](https://github.com/dallen68/PromoTube/pull/30#discussion_r450434236) with advantages detailed [here](https://dzone.com/articles/the-benefit-of-using-assertthat-over-other-assert).

also included in the latest commit in PR #30

File: src/main/java/com/google/step/youtube/YoutubeInfoScraper.java [hunk 2]
```diff
@@ -56,4 +57,22 @@ public YoutubeInfoScraper() {
...
+  * The optional will be empty if id is invalid or no items were found.
+  */
+   public Optional<List<PlaylistItem>> getPlaylistItems(String uploadId)
+    throws IOException {
+       PlaylistItemListResponse response = youTubeClient.playlistItems().list("snippet")
+            .setMaxResults(50L)
+            .setPlaylistId(uploadId)
+            .execute();
+        // getItems() return null when no items match the criteria (uploadId). 
+        if (response.getItems()==null || response.getItems().isEmpty()){
```
Comment (margaretchan) [inline src/main/java/com/google/step/youtube/YoutubeInfoScraper.java]: nit: space before {

**Review by margaretchan — CHANGES_REQUESTED** [review]

File: src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java (hunk 1 above)
Comment (josefj1519) [reply src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java]: Done.

File: src/main/java/com/google/step/youtube/YoutubeInfoScraper.java (hunk 2 above)
Comment (josefj1519) [reply src/main/java/com/google/step/youtube/YoutubeInfoScraper.java]: Done.

**Review by margaretchan — APPROVED** [review]

**Review by dallen68 — APPROVED** [review]

**Review by LiRq95 — APPROVED** [review]

**Pull request merged**